lms/djangoapps/courseware/courses.py

```diff
@@ -441,3 +445,47 @@ def get_problems_in_section(section):
                     problem_descriptors[unicode(component.location)] = component
 
     return problem_descriptors
+
+
+def get_entrance_exam_score(request, course):
```

Comment:
@asadiqbal08 -- please add a test to cover this method directly -- it is currently only being covered through the views tests.